week8/trivia/index.js

```diff
@@ -0,0 +1,24 @@
+let correct = document.querySelector('.correct');
```

Comment:
correct what? All things you learned in C apply here too. correct is not a good name for a variable since I don't know what it does and so on.